x/mongo/driver/operation_test.go

```diff
@@ -282,6 +282,72 @@ func TestOperation(t *testing.T) {
 			}
 		})
 	})
+	t.Run("calculateMaxTimeMS", func(t *testing.T) {
+		timeout := 5 * time.Second
+		maxTime := 2 * time.Second
+		negMaxTime := -2 * time.Second
+		shortRTT := 50 * time.Millisecond
+		longRTT := 10 * time.Second
```

Comment:
`shortRTT`, `longRTT`, and `timeout` are all used as input for `rtt90` in `(Operation).calculateMaxTimeMS`, [shouldn't these be given in nanoseconds for a truly accurate result](https://pkg.go.dev/time#:~:text=A%20Duration%20represents%20the%20elapsed%20time%20between%20two%20instants%20as%20an%20int64%20nanosecond%20count)?